### ISSUE TITLE
Oblivion + NVAC = broke

### ISSUE THREAD

**Issue by LostDragonist** [ModOrganizer2/modorganizer#1224]
Reported by Night Thastus on discord

MO 2.3.2
GOG Oblivion
OBSE
NVAC 7.5.1.0

Trying to launch the game with the above stuff results in a spinlock and the game never launches.  Note the recursion/looping in nvac.log.

[mo_interface (24).log](https://github.com/ModOrganizer2/modorganizer/files/5159967/mo_interface.24.log)
[usvfs-2020-09-02_03-30-37 (1).log](https://github.com/ModOrganizer2/modorganizer/files/5159968/usvfs-2020-09-02_03-30-37.1.log)
[nvac (1).log](https://github.com/ModOrganizer2/modorganizer/files/5159969/nvac.1.log)

**Comment by LostDragonist** [comment]
Something to note is that it worked once on initial install of everything and then not again (restarting PC did not help).  There might be some file that was created during the first run that broke all subsequent runs.

**Comment by FeuerTiger86** [comment]
Same for me. When I extract MO2 2.3.2 into a new folder, it works until I close the program. After that, the game doesn't start at all with NVAC enabled. No error messages or anything, and the NVAC log doesn't say anything.

It works with 2.2.2.1, though.

**Comment by Tyler799** [comment]
Some users who report this say it's related to a Windows 10 update. So for me, I was able to play with NVAC on 2.3.2 for a couple weeks, before some update happened and it stopped working.

**Comment by FeuerTiger86** [comment]
It's possible, I recently updated to 2004.

**Comment by JamesYeoman** [comment]
Oh wait this looks like the issue I've been having with #1228 
I'm getting the spinlocks, but it works for some reason when I launch via the launcher rather than directly via oblivion.exe

**Comment by JamesYeoman** [comment]
I've been able to reproduce with no mods enabled aside from nvac
[crashDumps.zip](https://github.com/ModOrganizer2/modorganizer/files/5197763/crashDumps.zip)
[mo_interface.log](https://gist.github.com/JamesYeoman/191dff434a337076c40faeeb8eacfa4b)
[usvfs.log](https://gist.github.com/JamesYeoman/43637da7987039f2530c482c6ec63b50)

**Comment by JamesYeoman** [comment]
What's really confusing is the fact that by launching the game via `OblivionLauncher.exe`, it works, and I have the obse force loading setting configured as seen in 1228

**Comment by JamesYeoman** [comment]
Is there any update on this bug?

**Comment by Al12rs** [comment]
No, I don't have a setup to reproduce the issue or time to set one up.

**Comment by JamesYeoman** [comment]
I reproduced it with a virtually completely clean  setup. You just need to set up OBSE, and then install the NVAC plugin

**Comment by Silarn** [comment]
I've actually run a pretty modded setup with NVAC a couple time. Oblivion still crashes sometimes because it's Oblivion, but I can load the game even with massive texture packs, multiple OBSE mods, and an ENB/ReShade preset.

That being said I'm running a pretty 'new' build which should be similar to the nightly builds in Discord. Maybe give those a try and see how it works. I'm not sure why the latest release would be different though, USVFS was not changed since then.

**Comment by JamesYeoman** [comment]
@Silarn  what OS are you rocking? And are you using the GOG version or the Steam version of Oblivion?

**Comment by Tyler799** [comment]
Just coming back about ~1.5-2 months later to report this is still an issue in 2.4.0 alpha2, Windows 10 build 19041.5722 (version 2004) and NVAC 7.5.1.0 (from 2016),

**Comment by Tyler799** [comment]
This *may* be fixed in the latest dev build, 2.4 Alpha 3.2. It is for me at least.

Though it broke a few other things for me, like the UI and being able to open older saves.

**Comment by WildfireDarkstar** [comment]
It's broken for me in 2.4 Alpha 3.3.

**Comment by Tyler799** [comment]
> 
> 
> It's broken for me in 2.4 Alpha 3.3.

Odd, it's working for me in alpha 3.3. What version of Windows are you on? I'm on 20H2.

**Comment by Al12rs** [comment]
Can anyone test 2.4.0alpha5+?

**Comment by Foxisnick** [comment]
Still is broken. Tested on 2.4.1RC2. It says it is running but nothing ever opens. The process eventually must be killed.

[mo_interface.log](https://github.com/ModOrganizer2/modorganizer/files/6137781/mo_interface.log)
[nvac.log](https://github.com/ModOrganizer2/modorganizer/files/6137782/nvac.log)
[usvfs-2021-03-14_22-26-24.log](https://github.com/ModOrganizer2/modorganizer/files/6137783/usvfs-2021-03-14_22-26-24.log)

**Comment by Cyber-Snake** [comment]
Can confirm that it's still broken even with V2.4.2 . Just switched to WIndows Insider build and now my Oblivion doesn't launch with NVAC enabled, process just gets stuck. Bug Fixes in v2.4.2rc1.1 patch notes mention Windows Insider improvements but this issue is still there.
